src/views/queries/responseCo2scope3.tsx

```diff
@@ -172,9 +172,24 @@ export default class Scope3CarbonConsumptionResponseTemplates {
     )
   }
 
+  private multFactor = (unit: 'ug' | 'mg' | 'g' | 'kg' | 'tonne'): number => {
+    switch (unit) {
+      case 'ug':
+        return 1e-9
+      case 'mg':
+        return 1e-6
+      case 'g':
+        return 1e-3
+      case 'kg':
+        return 1
+      case 'tonne':
+        return 1e3
+    }
+  }
+
   private reduceResponse = (query: CarbonEmbodimentRes['data']): number => {
     return (
-      query.mass +
+      query.mass * this.multFactor(query.unit) +
```

Comment:
where do we use query.mass? must have missed that one

Reply:
Quite a lot of places. This replaces `emissions` elsewhere